src/status_im2/common/scroll_page/view.cljs

```diff
@@ -41,54 +27,99 @@
+           (reanimated/set-shared-value y @scroll-height)
+           (reanimated/set-shared-value opacity-animation
+                                        (reanimated/with-timing (if (>= @scroll-height threshold) 1 0))))
+        [@scroll-height])
+       [:<>
+        [reanimated/blur-view
+         {:blur-amount   32
+          :blur-type     :xlight
+          :overlay-color (if platform/ios? colors/white-opa-70 :transparent)
+          :style         (reanimated/apply-animations-to-style
+                          {:transform [{:translateY translate-animation}]}
+                          style/blur-slider)}]
+        [rn/view {:style {:z-index 6 :margin-top (if platform/ios? 44 0)}}
+         [reanimated/view
+          {:style (reanimated/apply-animations-to-style
+                   {:opacity opacity-animation}
+                   style/sticky-header-title)}
+          [rn/image
+           {:source cover
+            :style  style/sticky-header-image}]
+          [quo/text
+           {:size   :paragraph-1
+            :weight :semi-bold
+            :style  {:line-height 21}}
+           name]]
+         [quo/page-nav
+          {:horizontal-description? true
+           :one-icon-align-left?    true
+           :align-mid?              false
+           :page-nav-color          :transparent
+           :mid-section             {:type            :text-with-description
+                                     :main-text       nil
+                                     :description-img nil}
+           :right-section-buttons   (:right-section-buttons page-nav)
+           :left-section            {:icon                  :i/close
+                                     :icon-background-color (icon-color)
+                                     :on-press              #(rf/dispatch [:navigate-back])}}]
+         (when sticky-header [sticky-header @scroll-height])]]))])
 
-(defn icon-size-fn
-  [scroll-height]
-  (->> (+ scroll-position-0 scroll-height)
-       (* (if platform/ios? 3 1))
-       (- max-image-size)
-       (max min-image-size)
-       (min max-image-size)))
+(defn display-picture
+  [scroll-height cover]
+  [:f>
+   (fn []
+     (let [input-range (if platform/ios? [-67 10] [0 150])
+           y           (reanimated/use-shared-value @scroll-height)
+           animation   (reanimated/interpolate y
+                                               input-range
+                                               [1.2 0.5]
+                                               {:extrapolateLeft  "clamp"
+                                                :extrapolateRight "clamp"})]
+       ;(rn/use-effect  #(reanimated/set-shared-value y @scroll-height)
+       ;               [@scroll-height])
+       (quo.react/effect! #(reanimated/set-shared-value y @scroll-height))
+       [reanimated/view
+        {:style (reanimated/apply-animations-to-style
+                 {:transform [{:scale animation}]}
+                 style/display-picture-container)}
```

Comment:
Ouch! I would definitely need to dig deeper to know why it fails like that. Thanks for sharing

Reply:
@ilmotta I figured it out. `useEffect` should return nothing or `undefined`, but for some reason, it was returning the value from the dependecy array which was `-44`. Fixed it by returning undefined from the hook function, but it would be better to update the implementation. Opened an issue: https://github.com/status-im/status-mobile/issues/14714

Reply:
You're absolutely correct. Thank you!